### PR TITLE
Refactor the GeoIP2 tests to use MaxMind test data

### PR DESCRIPTION
**Pull request by orf**
MaxMind publish small example databases that can be used for tests like these. This PR removes the need for a full install of MaxMind (which comes with licensing issues) in favour of their two city/country files published in the MaxMind-DB repository (https://github.com/maxmind/MaxMind-DB)

Closes https://github.com/django/django-docker-box/issues/14

### CODE REVIEW

File: django/contrib/gis/geoip2/base.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 from .resources import City, Country
 
+
```
Comment (iamdevvalecha) [inline django/contrib/gis/geoip2/base.py:12]: why this

**Review by iamdevvalecha — APPROVED** [review]

File: django/contrib/gis/geoip2/base.py (hunk 1 above)
Comment (orf) [reply django/contrib/gis/geoip2/base.py:12]: By defining the dictionary here the settings are loaded at import time. This means the `override_settings` decorator does not work, so we need to move this definition into the constructor.

**Comment by pope1ni** [comment]
Hi @orf. Ah, so I'd started on this too, but was looking into making the unit tests a bit more comprehensive, especially now that we can rely on the content of the database being fixed and unlikely to change frequently. I'll see if I can finish off what I started and we can combine.

**Comment by felixxm** [comment]
@orf Please send a new patch when you have time, thanks :star: .

**Comment by ngnpope** [comment]
@felixxm I'll try to resurrect what I had in the referenced PR. It was basically done, just needs rebasing and splitting up...

**Comment by felixxm** [comment]
> @felixxm I'll try to resurrect what I had in the referenced PR. It was basically done, just needs rebasing and splitting up...

Great, thanks :+1:

**Pull request closed without merging**